### PR TITLE
Feature/46 write board class

### PR DESCRIPTION
**Pull request by quirogas**

### CODE REVIEW

File: Model/src/GamePackage/Player.java [hunk 1]
```diff
@@ -1,269 +0,0 @@
-package GamePackage;
```
Comment (Dcbaas) [inline Model/src/GamePackage/Player.java]: Is this the file that we needed to delete?

File: res/board.txt [hunk 2]
```diff
@@ -0,0 +1,40 @@
+5,GO,200
```
Comment (Dcbaas) [inline res/board.txt]: Don't we want US version?

File: src/Model/BoardPackage/Board.java [hunk 3]
```diff
@@ -1,7 +1,93 @@
...
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
```
Comment (Dcbaas) [inline src/Model/BoardPackage/Board.java]: This class needs comments.

File: src/mainTest.java [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import Model.BoardPackage.Board;
+
+
+/**
+ * Created by unclear on 2/16/18.
+ */
```
Comment (Dcbaas) [inline src/mainTest.java]: Needs comments.

File: Model/src/GamePackage/Player.java (hunk 1 above)
Comment (quirogas) [reply Model/src/GamePackage/Player.java]: yes

File: res/board.txt (hunk 2 above)
Comment (quirogas) [reply res/board.txt]: no

**Comment by kernohad** [comment]
Changes requested were just comments, Comments were added. I am going to merge this so we can continue with Game class

**Pull request merged**